front/app/containers/Admin/projects/project/index.tsx

```diff
@@ -341,13 +338,29 @@ export class AdminProjectsProjectIndex extends PureComponent<
       },
       tabs: !isNilOrError(project) ? this.getTabs(project.id, project) : [],
     };
+
+    let numberIdeationPhases = 0;
+    let ideationPhase;
+
     if (!isNilOrError(project) && phases !== undefined) {
       const inputTerm = getInputTerm(
         project?.attributes.process_type,
         project,
         phases
       );
 
+      if (!isNilOrError(phases)) {
+        phases.map((phase) => {
+          if (
+            getMethodConfig(phase.attributes.participation_method)
+              .showInputManager
+          ) {
+            numberIdeationPhases++;
+            ideationPhase = phase;
+          }
```

Comment:
This seemed rather specific to have in a separate util (finding both number of ideation phases as well as the phase itself), but let me know if you think I should handle this in a function in the ParticipationMethodUtils instead.